events-api/src/modules/auth/auth.controller.ts

```diff
@@ -26,4 +34,9 @@ export class AuthController {
       data: authData,
     };
   }
+
+  @Get()
+  async getAllUsers() {
```

Comment:
It's not a responsibility of auth module to return all users. Move it to users service for example